lib/db/keyer_test.go

```diff
@@ -46,36 +46,6 @@ func TestDeviceKey(t *testing.T) {
 	}
 }
 
-func TestGlobalKey(t *testing.T) {
-	fld := []byte("folder6789012345678901234567890123456789012345678901234567890123")
-	name := []byte("name")
-
-	db := NewLowlevel(backend.OpenMemory())
-	defer db.Close()
-
-	key, err := db.keyer.GenerateGlobalVersionKey(nil, fld, name)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	fld2, ok := db.keyer.FolderFromGlobalVersionKey(key)
-	if !ok {
-		t.Error("should have been found")
-	}
-	if !bytes.Equal(fld2, fld) {
-		t.Errorf("wrong folder %q != %q", fld2, fld)
-	}
-	name2 := db.keyer.NameFromGlobalVersionKey(key)
```

Comment:
I wouldn't remove the whole test - this function is still in use.

Reply:
Restored.